### ISSUE TITLE
Updated Google oauth example

### ISSUE THREAD

**Issue by Joabesv** [fastify/fastify-oauth2#210]
### Prerequisites

- [X] I have written a descriptive issue title
- [X] I have searched existing issues to ensure the issue has not already been raised

### Issue

Hi there! i'm finding really hard to setup this package with googleOauth and the example doesn't see of much help since it looks outdated.

was wondering if we can get an updated version of the google auth example?

thanks in advance.

**Comment by climba03003** [comment]
The example looking just fine.
Can you point me which part of it looks outdated?

**Comment by Joabesv** [comment]
Yeah, after some days of debugging.  the error was on my  credentials, tested again the example and things worked fine.

Will be closing this issue now.
